### DISCUSSION TITLE
fs-ts for typescript problems

### DISCUSSION THREAD

**Discussion by getTobiasNielsen** [codewars/codewars.com#2922] [Ideas]
Would people be interested in having [fp-ts](https://github.com/gcanti/fp-ts) as a package for ts?

Current package list: https://docs.codewars.com/languages/typescript#packages

**Comment by hobovsky** [comment]
Just create a package request at [`codewars/runner` repository](https://github.com/codewars/runner/issues) and admins will add it if they think it's worth it.
